pkg/helm/actions/install_chart_test.go

```diff
@@ -1,16 +1,20 @@
 package actions
 
 import (
+	"fmt"
 	"io/ioutil"
 	"testing"
+	tm "time"
```

Comment:
I would prefer if we keep the default module name, that makes it easier to understand in the code.

```suggestion
	"time"
```